railties/lib/rails/application/configuration.rb

```diff
@@ -34,6 +34,7 @@ def initialize(*)
         @filter_redirect                         = []
         @helpers_paths                           = []
         @hosts                                   = Array(([".localhost", IPAddr.new("0.0.0.0/0"), IPAddr.new("::/0")] if Rails.env.development?))
+        @hosts.concat(ENV["RAILS_DEVELOPMENT_HOSTS"].split(",")) if ENV["RAILS_DEVELOPMENT_HOSTS"] && Rails.env.development?
```

Comment:
```suggestion
        @hosts.concat(ENV["RAILS_DEVELOPMENT_HOSTS"].to_s.split(",")) if Rails.env.development?
```
seems less wordy to me.